tensorflow/python/autograph/operators/py_builtins_test.py

```diff
@@ -137,6 +137,11 @@ def test_range_tensor_empty_range(self):
       r = py_builtins.range_(5, constant_op.constant(2))
       self.assertAllEqual(self.evaluate(r), [])
 
+  def test_enumerate(self):
```

Comment:
Please also include a test that uses an actual Dataset. You can copy the model of `test_len_dynamic_shape` for running graph ops.

Reply:
I'm following [this one](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/data/kernel_tests/enumerate_test.py). It should be okay right?

Reply:
That test is uses a special base test class, so it might not port directly - check if the test passes, and if it doesn't, I have a suggestion below.